src/package/nuspec/Microsoft.NET.Test.Sdk.targets

```diff
@@ -41,6 +41,15 @@
   <PropertyGroup>
     <DebugType Condition="'$(TargetFrameworkIdentifier)' != '.NETCoreApp'">Full</DebugType>
   </PropertyGroup>
+ 
+  <!--
+     Generate config file for test project targeting .NET Framework. This config file has have binding redirect which
+	 is needed at time of running tests.
+   -->
+  <PropertyGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework'">
+    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
+    <GenerateBindingRedirectsOutputType>true</GenerateBindingRedirectsOutputType>
```

Comment:
This is required due to this piece of code
https://github.com/Microsoft/msbuild/blob/dd5e8bc3f86ac98bd77d8971b00a6ad14f122f1a/src/XMakeTasks/Microsoft.Common.CurrentVersion.targets#L2027